.eslintrc.js

```diff
@@ -79,6 +79,19 @@ module.exports = {
                 es2017: true
             }
         },
+        {
```

Comment:
note: https://github.com/mozilla/bedrock/pull/11111/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5
has the bigger linting change and should probably merge first, conflicts incoming either way!